assembl/static2/webpack.config.js

```diff
@@ -96,10 +98,6 @@ module.exports = {
           exclude: /node_modules/,
           use: 'graphql-tag/loader'
         },
-        {
```

Comment:
why don't we need this anymore?

Reply:
json-loader ? webpack handles json import natively since webpack 2